wallet/subnet/primary/examples/disable-subnet-validator/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/ava-labs/avalanchego/genesis"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+)
+
+func main() {
+	key := genesis.EWOQKey
+	uri := primary.LocalAPIURI
+	kc := secp256k1fx.NewKeychain(key)
+	validationID := ids.FromStringOrPanic("9FAftNgNBrzHUMMApsSyV6RcFiL9UmCbvsCu28xdLV2mQ7CMo")
```

Comment:
where is this ID taken from?

Reply:
This is just an example, so running this against an actual network would likely need to change this value.

This value was taken from the result of either running the `convert-subnet` example or the `register-subnet-validator` example.